cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/controllers/CatsSqlAdminController.kt

```diff
@@ -102,7 +93,39 @@ class CatsSqlAdminController(
     return CleanTablesResult(tableCount = tablesDropped.size, tables = tablesDropped)
   }
 
-  private fun validateParams(currentNamespace: String?, targetNamespace: String) {
+  @PutMapping(path = ["drop_version/{version}"])
```

Comment:
> Also to be honest I can't find how to run the commands in here, do you know how to?

I might be misunderstanding your question from [this comment](https://github.com/spinnaker/clouddriver/pull/4755#issuecomment-663562679) but I think you should be able to just curl a PUT request against `{your_clouddriver}/admin/db/drop_version/{version}` to test this. Sorry if that is not in fact what you were asking!

Reply:
This was exactly what I was looking for, thank you! Though when I try and run it I get an unauthorized error. Any idea on what else I need to run these endpoints?

Command I ran: ```curl -X PUT http://localhost:7002/admin/db/drop_version/V1```

Response: ```{"error":"Internal Server Error","message":"Unauthorized","status":500,"timestamp":"2020-07-28T21:30:38.334+00:00"}```

Reply:
Likely because it is checking whether you are an [admin](https://spinnaker.io/setup/security/admin/#manually-add-configuration-in-fiat)? If you don't already have fiat configured, could also just test it on your LocalGit-deployed dev spinnaker with the perm checks commented out.

Reply:
Thanks, I don't have fiat configured so I went with commenting out the permission check in my localgit deployed spinnaker to test and it works.